src/Controls/tests/DeviceTests/Elements/ContextFlyout/ContextFlyoutTests.Windows.cs

```diff
@@ -13,7 +13,6 @@ public partial class ContextFlyoutTests : ControlsHandlerTestBase
 		[Fact(DisplayName = "Context flyout creates expected WinUI elements")]
 		public async Task ContextFlyoutCreatesExpectedWinUIElements()
 		{
-			SetupBuilder();
```

Comment:
Another case that I am cleaning up is the proliferation of the `SetupBuilder()` calls. Using the nice (and existing) `ConfigureBuilder` method on the base, we can set up for all the tests in the class.